easybuild/easyconfigs/m/MATIO/MATIO-1.5.21-GCCcore-10.2.0.eb

```diff
@@ -0,0 +1,34 @@
+##
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+#
+# Copyright:: Copyright 2014-2015 The Cyprus Institute
+# Authors:: Thekla Loizou <t.loizou@cyi.ac.cy>
+# License:: MIT/GPL
+#
+##
+easyblock = 'ConfigureMake'
+
+name = 'MATIO'
+version = '1.5.21'
+
+homepage = 'https://sourceforge.net/projects/matio/'
+description = """matio is an C library for reading and writing Matlab MAT files."""
+
+toolchain = {'name': 'GCCcore', 'version': '10.2.0'}
+
+source_urls = [SOURCEFORGE_SOURCE]
+sources = [SOURCELOWER_ZIP]
+checksums = ['bf1808a20412534a7c9e69c6eb0d2612']
```

Comment:
not a sha256sum?

Reply:
Whoops, that looks like a copy/paste error.

It should be:

2ac1d5406e51f2bdfaa545a535d6dbf6347be89891a06323961d7a2acbf37be9